README.md

```diff
@@ -60,6 +60,16 @@ EOF
 sudo dnf install lavinmq
 ```
 
+### Archlinux
+
+The package is available on [AUR](https://aur.archlinux.org/packages/lavinmq).
+
+```sh
+yay -S lavinmq
```

Comment:
Really nice with and AUR package, thank you! 

I'm also on Arch but I'm using `paru` instead of `yay`. I think we should either have both here or none of them. 
Linking to the AUR page should be enough and then everyone can figure out how to install this using their AUR package manager.